### DISCUSSION TITLE
nivatech ntc 7601 driver

### DISCUSSION THREAD

**Discussion by kodmanyagha** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#79] [General]
I bought a usb wifi adapter which model is "nivatech ntc 7601" and probably this driver isn't compatible with this. If you handle this problem then I will be very grateful. Thanks.

**Comment by chris1111** [comment]
> I bought a usb wifi adapter which model is "nivatech ntc 7601" and probably this driver isn't compatible with this. If you handle this problem then I will be very grateful. Thanks.


Please try this [Mediatek Drivers](https://github.com/chris1111/D-LinkUtility-Package)
